src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -138,7 +139,13 @@ private void addDocument(SortableBundleItem item) throws IOException {
             final PDDocumentOutline newDocOutline = newDoc.getDocumentCatalog().getDocumentOutline();
             newDoc.getDocumentCatalog().setDocumentOutline(null);
 
-            merger.appendDocument(document, newDoc);
+            try {
+                merger.appendDocument(document, newDoc);
+            }
+            catch (IndexOutOfBoundsException e) {
+                newDoc.getDocumentCatalog().setStructureTreeRoot(new PDStructureTreeRoot());
```

Comment:
Sorry to drag this out, but could we log the fact that we've done this. Just something `log.info("Corrected PDF structure of " + filename)` or something similar